pints/_log_likelihoods.py

```diff
@@ -237,6 +237,228 @@ def __call__(self, x):
         )
 
 
+class CombinedGaussianLogLikelihood(pints.ProblemLogLikelihood):
+    r"""
+    Calculates the log-likelihood for either a
```

Comment:
Would try shortening this a bit and leaving the `:math:` to the second paragraph.
E.g. "Log-likelihood that assumes a mixture of Gaussian noise and Gaussian heteroscedastic noise.". And then the 2nd paragraph gives math, explanation of heteroscedastic, single and multi, etc.

Reply:
For example, does this generalise to some kind of polynomial normal + normal * model + normal * model^2 + ... etc.?
Needs more specific name!

Reply:
Done